src/main/java/org/broadinstitute/hellbender/tools/gvs/common/GQStateEnum.java

```diff
@@ -10,6 +10,8 @@ public enum GQStateEnum {
     FORTY("4", 40, 4),
     FIFTY("5", 50, 5),
     SIXTY("6", 60, 6),
+    // NOTE: MISSING is no longer used (it is now being written as ZERO, *unless* we are dropping ref_blocks with that state.
```

Comment:
```suggestion
    // NOTE: MISSING is no longer used (it is now being written as ZERO, *unless* we are dropping ref_blocks with that state.)
```

Reply:
omg we still have "v" in here lol
I think that is SO OLD we would break in 1000 other ways before it complained about the enum